tests/ui/hygiene/extern-prelude-from-opaque-fail-2018.stderr

```diff
@@ -25,8 +25,8 @@ LL | a!();
    | ---- in this macro invocation
    |
    = help: consider importing one of these items:
-           std::mem
            core::mem
+           std::mem
```

Comment:
this doesn't look intended

Reply:
The message line and the suggestion lines were inconsistent, now they are in the same order.